python/GangaDirac/Lib/RTHandlers/ExeDiracRTHandler.py

```diff
@@ -42,31 +43,52 @@ def prepare(self, app, appsubconfig, appmasterconfig, jobmasterconfig):
         job = stripProxy(app).getJobObject()
         outputfiles = [this_file for this_file in job.outputfiles if isType(this_file, DiracFile)]
 
-        commandline = app.exe
+        commandline = []
+        commandline.append(app.exe)
         if isType(app.exe, File):
             #logger.info("app: %s" % str(app.exe.name))
             #fileName = os.path.join(get_share_path(app), os.path.basename(app.exe.name))
             #logger.info("EXE: %s" % str(fileName))
             #inputsandbox.append(File(name=fileName))
             inputsandbox.append(app.exe)
-            commandline = os.path.basename(app.exe.name)
-        commandline += ' '
-        commandline += ' '.join([str(arg) for arg in app.args])
+            commandline[0]=os.path.join('.', os.path.basename(app.exe.name))
+        commandline.extend([str(arg) for arg in app.args])
         logger.debug('Command line: %s: ', commandline)
 
         #exe_script_path = os.path.join(job.getInputWorkspace().getPath(), "exe-script.py")
         exe_script_name = 'exe-script.py'
 
+        logger.info("Setting Command to be: '%s'" % repr(commandline))
+
         inputsandbox.append(FileBuffer(name=exe_script_name,
                             contents=script_generator(exe_script_template(),
                                                     #remove_unreplaced = False,
                                                     # ,
-                                                    COMMAND=commandline,
+                                                    COMMAND=repr(commandline),
                                                     OUTPUTFILESINJECTEDCODE = getWNCodeForOutputPostprocessing(job, '    ')
                                                     ),
                                        executable=True))
 
-        dirac_outputfiles = dirac_outputfile_jdl(outputfiles)
+        contents=script_generator(exe_script_template(), COMMAND=repr(commandline),
+                                    OUTPUTFILESINJECTEDCODE = getWNCodeForOutputPostprocessing(job, '    ')
+                                    )
+
+        #logger.info("Script is: %s" % str(contents))
+
+        from os.path import abspath, expanduser
+
+        for _file in job.inputfiles:
```

Comment:
PEP 8 suggests leading underscores only for internal names. Trailing underscores should be used to avoid conflicts with Python names. Otherwise, `f` would do.